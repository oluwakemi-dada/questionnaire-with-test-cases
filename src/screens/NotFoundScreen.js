import React from 'react';
import { Link } from 'react-router-dom';
import styled from 'styled-components';
import { Wrapper } from '../components/form/FormResources';

const Content = styled.div`
  margin-top: 2rem;

  p {
    margin-top: 2rem;
    color: #5e55ef;
    font-weight: 500;
  }
`;

const NotFoundScreen = () => {
  return (
    <Wrapper>
      <Content>
        <h2>Sorry, this page does not exist ğŸ˜¥</h2>
        <Link to='/'>
          <p>Return home ğŸš€</p>
        </Link>
      </Content>
    </Wrapper>
  );
};

export default NotFoundScreen;
