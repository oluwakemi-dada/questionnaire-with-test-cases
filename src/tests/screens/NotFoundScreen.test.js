import React from 'react';
import { shallow } from 'enzyme';
import NotFoundScreen from '../../screens/NotFoundScreen';

test('should render NotFoundScreen correctly', () => {
  const wrapper = shallow(<NotFoundScreen />);
  expect(wrapper).toMatchSnapshot();
});
