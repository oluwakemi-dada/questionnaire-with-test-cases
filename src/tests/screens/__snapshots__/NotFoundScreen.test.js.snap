// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFoundScreen correctly 1`] = `
<styled.div>
  <styled.div>
    <h2>
      Sorry, this page does not exist ðŸ˜¥
    </h2>
    <Link
      to="/"
    >
      <p>
        Return home ðŸš€
      </p>
    </Link>
  </styled.div>
</styled.div>
`;
